doc/source/cluster/kubernetes/user-guides/logging.md

```diff
@@ -242,6 +244,118 @@ Learn how to write filters in this [filter pattern syntax doc](https://docs.aws.
 [KubDoc]: https://kubernetes.io/docs/concepts/cluster-administration/logging/
 [ConfigLink]: https://raw.githubusercontent.com/ray-project/ray/releases/2.4.0/doc/source/cluster/kubernetes/configs/ray-cluster.log.yaml
 
+### Method 2: Grafana Loki
+
+#### Deploy Loki monolithic mode:
+
+Loki’s Helm chart supports three deployment methods to fit different scalability and performance needs: Monolithic, Simple Scalable, and Microservices. Here, we will demonstrate the monolithic method. For details on each deployment mode, see the [Loki deployment](https://grafana.com/docs/loki/latest/get-started/deployment-modes/) modes documentation.
+
+Deploy the Loki deployment with the [Helm chart repository](https://github.com/grafana/loki/tree/main/production/helm/loki).
+
+```shell
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+
+# Install Loki with single replica mode
+helm install loki grafana/loki -f https://raw.githubusercontent.com/grafana/loki/refs/heads/main/production/helm/loki/single-binary-values.yaml
+```
+
+#### Configure log processing:
+
+Create a `fluent-bit-config.yaml` file which configures Fluent Bit to:
+
+* Tail log files from Kubernetes containers.
+* Parse multi-line logs for Docker and CRI (Container Runtime Interface) formats.
+* Enrich logs with Kubernetes metadata such as namespace, pod, and container names.
+* Send the logs to Loki for centralized storage and querying.
+```{literalinclude} ../configs/loki.log.yaml
+:language: yaml
+:start-after: Fluent Bit Config
+:end-before: ---
+```
+
+A few notes on the above config:
+
+* Inputs: The `tail` input reads log files from `/var/log/containers/*.log`, with `multiline.parser` to handle complex log messages across multiple lines.
+* Filters: The `kubernetes` filter adds metadata like namespace, pod, and container names to each log, enabling more efficient log management and querying in Loki.
+* Outputs: The `loki` output block specifies Loki as the target. The `Host` and `Port` define the Loki service endpoint, and `Labels` add metadata for easier querying in Grafana. Additionally, `tenant_id` allows for multi-tenancy if required by the Loki setup.
+
+Deploy the Fluent Bit deployment with the [Helm chart repository](https://github.com/fluent/helm-charts/tree/main/charts/fluent-bit).
+
+```shell
+helm repo add fluent https://fluent.github.io/helm-charts
+helm repo update
+
+helm install fluent-bit fluent/fluent-bit -f fluent-bit-config.yaml
+```
+
+#### Deploy a KubeRay operator and RayCluster
+
+Deploy the KubeRay operator and RayCluster with the [Helm chart repository](https://github.com/ray-project/kuberay-helm).
+
+```shell
+helm repo add kuberay https://ray-project.github.io/kuberay-helm/
```

Comment:
For the installation of KubeRay, we typically refer to the "RayCluster Quickstart" instead. For example, see: https://docs.ray.io/en/latest/cluster/kubernetes/getting-started/rayjob-quick-start.html#step-2-install-the-kuberay-operator. Keeping the "RayCluster Quickstart" as the single source of truth ensures that we don't need to update "1.2.2" to "1.3.0" in multiple documents when releasing a new KubeRay version.